signature-view/src/main/java/se/warting/signatureview/views/SignaturePad.kt

```diff
@@ -162,24 +162,28 @@ class SignaturePad(context: Context, attrs: AttributeSet?) : View(context, attrs
         val didDoubleClick = doubleClickGestureDetector.onTouchEvent(event)
         if (!isEnabled || didDoubleClick) return false
 
+        // Validate coordinates to prevent NaN/infinite values from entering the system
+        val x = if (event.x.isNaN() || event.x.isInfinite()) 0f else event.x
+        val y = if (event.y.isNaN() || event.y.isInfinite()) 0f else event.y
```

Comment:
instead of returning 0f this could crash, we really want to know where this is coming from